src/Resources/views/default/edit.html.twig

```diff
@@ -1,3 +1,10 @@
+{% set _select2_locales = ['ar', 'az', 'bg', 'ca', 'cs', 'da', 'de', 'el', 'en', 'es', 'et', 'eu', 'fa', 'fi', 'fr', 'gl', 'he', 'hi', 'hr', 'hu', 'id', 'is', 'it', 'ja', 'km', 'ko', 'lt', 'lv', 'mk', 'ms', 'nb', 'nl', 'pl', 'pt-BR', 'pt', 'ro', 'ru', 'sk', 'sr-Cyrl', 'sr', 'sv', 'th', 'tr', 'uk', 'vi', 'zh-CN', 'zh-TW'] %}
```

Comment:
Add this straight to the layout : every page will benefit of the entity-form-modal (not only related to the autocomplete option).

No more need to modify `edit.html.twig` nor `new.html.twig`.

Reply:
Send changes would be solved?